src/analytics/Properties.tsx

```diff
@@ -1257,13 +1257,15 @@ interface SwapEventsProperties {
       fromTokenBalance: string
       swapExecuteTxId: string
       swapApproveTxId: string
+      estimatedUsdValue?: number
```

Comment:
nit: Do you think it makes sense to include this field in the `SwapQuoteEvent`?

Reply:
yeah, probably.  I was going to chat with you about clarifying some of the other fields (e.g., `.usdValue`) before adding this one to `SwapQuoteEvent` (and by implication `swap_review_submit`).